packages/react/__tests__/exports.ts

```diff
@@ -17,6 +17,7 @@ describe('@aws-amplify/ui-react', () => {
           "defaultDarkModeOverride",
           "defaultTheme",
           "translations",
+          "stylePropsToThemeKeys",
```

Comment:
Hmm, I don't think we want to expose this as an external API

Reply:
this has been removed